packages/babel-plugin-transform-runtime/test/fixtures/options.json

```diff
@@ -1,3 +1,4 @@
 {
-  "externalHelpers": false
+  "externalHelpers": false,
+  "targets": { "browsers": "ie 6, ios 4" }
```

Comment:
Added `ios 4` here because IE 5.5 has `Array.map` but not `ios 4`. Note that `browserslist` does not support IE 5.